src/components/SavedProjectPlan/ProjectPlanCard.jsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Divider,
+  Flex,
+  Heading,
+} from "@chakra-ui/react";
+import React from "react";
+import StandardCard from "./StandardCard";
+
+const ProjectPlanCard = () => {
+  return (
+    <Flex>
+      <Card boxShadow="md" p="6" rounded="mlgd" bg="white" marginBottom="10">
+        <CardHeader display="flex" justifyContent="space-between">
+          <Box display="flex">
+            <Heading size="xl">Recent Project Plan</Heading>
+            <Button
+              as="a"
+              background="#6D6E70"
+              borderRadius="3xl"
+              color="white"
+              marginLeft="15"
+            >
+              Download
+            </Button>
+            <Button
+              as="a"
+              background="#6D6E70"
+              borderRadius="3xl"
+              color="white"
+              marginLeft="15"
+            >
+              Print to PDF
+            </Button>
+          </Box>
+          <Box>
+            <Button
+              as="a"
+              background="#B90000"
+              borderRadius="3xl"
+              color="white"
+            >
+              Remove from Saved Project Plans
+            </Button>
+          </Box>
+        </CardHeader>
+        <CardBody>
+          <Flex justifyContent="space-between" marginRight="15em">
+            <StandardCard />
```

Comment:
If you figure out how to do the default props thing, multiply the default props by 3 so you have array of three default props, and then use the map function to then render out the standard card and divider.

I.e `[arrayOfDefaultProps].map(defaultProps => {
  return <StandardCard title="defaultProps.title" imageUrls="defaultProps.imageUrls>
})` + add the divider. you get the idea.